### PR TITLE
TELCODOCS-1305 - Adding 4.14 ZTP software versions

### PR DESCRIPTION
**Pull request by aireilly**
Adding ZTP solution software versions for 4.14.

Version(s):
main, enterprise-4.14

Issue:
https://issues.redhat.com/browse/TELCODOCS-1305

Link to docs preview:
https://66252--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster#ztp-gitops-ztp-max-spoke-clusters_ztp-preparing-the-hub-cluster

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by aireilly** [comment]
@mpmaruthu can you PTAL for QE review.

**Review by jldohmann — APPROVED** [review]
LGTM

**Review by sabbir-47 — CHANGES_REQUESTED** [review]
@aireilly for CLO operator we need to downgrade the version from 5.8 -> 5.7

File: modules/ztp-telco-ran-software-versions.adoc [hunk 1]
```diff
@@ -2,30 +2,55 @@
...
+|Component
 |Software version
 
-|Hub cluster {product-title} version
-|4.13
+|RAN DU cluster version
+|4.14
+
+|Cluster Logging Operator
+|5.8
```
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/ztp-telco-ran-software-versions.adoc]: Updated

**Comment by aireilly** [comment]
@mpmaruthu Can you QE review this one too please.

**Comment by mpmaruthu** [comment]
Telco QE: Looks good to me.
/Peri M

**Pull request merged**